app/controllers/aquariums_controller.rb

```diff
@@ -2,4 +2,8 @@ class AquariumsController < ApplicationController
   def index
     @aquariums = Aquarium.all
   end
+
+  def show
+    @aquarium = Aquarium.find(params[:id])
```

Comment:
Can you explain this part to me?

Reply:
This is an active record method, basically specifying that the params take an argument of `:id` so it can be found for the Aquariums. That said, I'm still a little fuzzy on if it's referring to the Aquarium class or table.